libs/util/src/lib/give-zero-padding.ts

```diff
@@ -0,0 +1,4 @@
+export const giveZeroPadding = (num: string, size: number): string => {
+  const s = num + '';
+  return s.padStart(size, '0');
```

Comment:
You can simplify this as follows:
`${num}`.padStart(size, '0');`

Reply:
Done a7a004fa2658f128a28d7d4735d943f10b8a030d